### PR TITLE
added an image title, background and subheading

### PR DESCRIPTION
**Pull request by KS389-ops**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  } 
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+  }
+button {
+    background-color: #baba00; /* Blue */;
```
Comment (samyou-softwire) [inline src/index.css]: I think this comment is out of date
```suggestion
    background-color: #baba00; /* Yellow */;
```

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    font-size:  40px;
+    border-radius: 50%;
+        text-align:center;
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        width: 40%;
```
Comment (samyou-softwire) [inline src/index.css]:
```suggestion
    text-align:center;
    display: block;
    margin-left: auto;
    margin-right: auto;
    width: 40%;
```

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    margin-right: auto;
+    width: 40%;
+  }
+body {
+    background-image: url("assets/tetris background.png");
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+  }
```
Comment (samyou-softwire) [inline src/index.css]:
```suggestion
} 
```

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
+h1 {
+    color: rgb(3, 148, 35);
+    font-style: oblique;text-shadow:color-mix(in hwb longer hue, color percentage, color percentage);
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
+  }
```
Comment (samyou-softwire) [inline src/index.css]:
```suggestion
}
```

File: src/index.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
+h1 {
+    color: rgb(3, 148, 35);
+    font-style: oblique;text-shadow:color-mix(in hwb longer hue, color percentage, color percentage);
```
Comment (samyou-softwire) [inline src/index.css]:
```suggestion
    font-style: oblique;
    text-shadow:color-mix(in hwb longer hue, color percentage, color percentage);
```

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+  <head>
+    <link href="index.css" rel="stylesheet" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>TETRIS</title>
+  </head>
+
+  <body>
+    <img src="assets/tetris title image.png" alt="TETRIS" style="width:400px;height:200px;">
```
Comment (samyou-softwire) [inline src/index.html]: `style=` is an alternate way of adding styling to an element, though we should probably keep all our styling in index.css. Are you able to move this?

**Review by samyou-softwire — COMMENTED** [review]
Nice work :), a couple formatting tweaks to make.

Perhaps also more broadly, worth keeping track of where these images have come from, so we can ensure they're properly credited!

File: src/index.html (hunk 6 above)
Comment (KS389-ops) [reply src/index.html]: yes, thank you!

**Pull request merged**